Makefile

```
@@ -72,10 +180,102 @@ else
+#   make bench SEED7_BENCH_DIR=/path/to/files   # use a custom directory
+#
+# Targets:
+#   bench-check   verify SEED7_BENCH_DIR contains at least one matching file
+#   bench         run the benchmark on all .sd7 and .s7i files
+
+# Directory that contains the .sd7 / .s7i files to benchmark.
+SEED7_BENCH_DIR ?= /usr/local/lib/seed7
+
+.PHONY: bench bench-check
+
+bench-check:
```

checkmake (0.3.2)

[warning] Target body for "bench-check" exceeds allowed length of 5 lines (6).

(maxbodylength)